temporal-sdk/src/main/java/io/temporal/worker/Worker.java

```
@@ -384,16 +384,12 @@ public String toString() {
    * @throws Exception if replay failed for any reason
    */
   public void replayWorkflowExecution(WorkflowExecutionHistory history) throws Exception {
-    try {
-      workflowWorker.queryWorkflowExecution(
-          history,
-          WorkflowClient.QUERY_TYPE_REPLAY_ONLY,
-          String.class,
-          String.class,
-          new Object[] {});
-    } catch (Exception e) {
-
-    }
```

Wow, not sure how a such ignore all exceptions cath ended up in the code :(.